src/contract.rs

```diff
@@ -71,6 +72,7 @@ pub fn handle<S: Storage, A: Api, Q: Querier>(
         HandleMsg::Transfer { recipient, amount, .. } => try_transfer(deps, env, &recipient, amount),
         // todo Send
         HandleMsg::Burn { amount, .. } => try_burn(deps, env, amount),
+        HandleMsg::Mint { amount, address } => try_mint(deps, env, address, amount),
         // todo RegisterReceive
```

Comment:
Put all the "mint" related messages in their own section under the other messages